staging/src/k8s.io/pod-security-admission/test/testdata/baseline/v1.0/pass/sysctls1.yaml

```diff
@@ -21,3 +21,5 @@ spec:
       value: 1 0
     - name: net.ipv4.ip_unprivileged_port_start
       value: "1024"
+    - name: net.ipv4.ip_local_reserved_ports
```

Comment:
past versioned fixtures for the sysctl check are a good example of what we don't want to change... the 1.0 fixture should remain as-is, for example